hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: patch
```

Comment:
This *could* be a patch release because it doesn't touch the public API, but I reckon it's a shiny enough capability that it deserves to be a minor release - very few PBT tools have swarm testing built in!